awx/main/tests/data/inventory/plugins/azure_rm/files/azure_rm.yml

```diff
@@ -0,0 +1,35 @@
+conditional_groups:
+  azure: true
+default_host_filters: []
+exclude_host_filters:
+- resource_group not in ['foo_resources', 'bar_resources']
+- location not in ['southcentralus', 'westus']
+hostvar_expressions:
+  ansible_host: private_ipv4_addresses | json_query("[0]")
```

Comment:
`private_ipv4_addresses[0]` will net the same result and negate the need for `json_query`